db/migrate/20170217134810_add_foreign_key_for_unpublishings.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyForUnpublishings < ActiveRecord::Migration[5.0]
+  def change
+    add_foreign_key :unpublishings, :editions
```

Comment:
yay 👍  does this take long to run? 

what do you think about having a delete: cascade on unpublishings?